node/actors/bft/src/testonly/ut_harness.rs

```diff
@@ -83,8 +135,8 @@ impl UTHarness {
         &self.consensus.inner.secret_key
     }
 
-    pub(crate) fn owner_as_view_leader(&self) -> ViewNumber {
-        let mut view = self.current_replica_view();
+    pub(crate) fn owner_as_view_leader_next(&self) -> ViewNumber {
```

Comment:
nit: IMO the "next" suffix seems to suggest at the very least that the returned view will be AFTER the current view.

Reply:
I wanted to imply that it's not _beyond_ the next one. 
But perhaps there will be a use case where the current is to be skipped, so I made it explicit with a longer name: `owner_as_view_leader_current_or_next`.